manifests/database/mysql.pp

```diff
@@ -23,6 +23,7 @@
   $database_user        = '',
   $database_password    = '',
   $database_host        = '',
+  $database_port        = 3306,
```

Comment:
please add a datatype here and some inline documentation (I know that the other params aren't document at the moment :()
I suggest [Stdlib::Port::Unprivileged](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/port/unprivileged.pp)
```suggestion
  Stdlib::Port::Unprivileged $database_port = 3306,
```

Reply:
Thanks! What do you mean by "inline documentation"? Like next to the parameter?

Reply:
in the header of the file, like in https://github.com/voxpupuli/puppet-zabbix/blob/master/manifests/template.pp :)

Reply:
Do you want me to document them all as we are here?

Reply:
if you want to do that, that would be awesome. In that caee please provide them as puppet-strings style